download.md

```diff
@@ -43,6 +43,21 @@ The latest release of Apache Zeppelin is **0.7.2**.
     [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2.tgz.md5),
     [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2.tgz.sha512))
 
+# Using the official docker image 
+
+Make sure that [docker](https://www.docker.com/community-edition) is installed in your local machine.  
+
+Use this command to launch Apache Zeppelin in a container. 
+
+```bash
+docker run -p 8080:8080 --rm --name zeppelin apache/zeppelin:0.7.2
+```
+
+To persist `logs` and `notebook` directories in local machine, Use the [volumn](https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v-read-only) option for docker container.
```

Comment:
it's minor; could you start with a lowercase letter after the comma? 
`, Use` -> `, use`